server/src/internalClusterTest/java/org/opensearch/indices/IndicesRequestCacheIT.java

```diff
@@ -1329,11 +1345,26 @@ private void setupIndex(Client client, String index) throws Exception {
                         .put(IndicesRequestCache.INDEX_CACHE_REQUEST_ENABLED_SETTING.getKey(), true)
                         .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
                         .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                        // Disable index refreshing to avoid cache being invalidated mid-test
+                        .put(IndexSettings.INDEX_REFRESH_INTERVAL_SETTING.getKey(), TimeValue.timeValueMillis(-1))
+                        // settings to reduce the chances of background segment merges invalidating the cache
+                        .put(TieredMergePolicyProvider.INDEX_MERGE_POLICY_SEGMENTS_PER_TIER_SETTING.getKey(), 1000d)
+                        .put(TieredMergePolicyProvider.INDEX_MERGE_POLICY_MAX_MERGE_AT_ONCE_SETTING.getKey(), 2)
+                        .put(TieredMergePolicyProvider.INDEX_MERGE_POLICY_MAX_MERGED_SEGMENT_SETTING.getKey(), "100gb")
+                        .put(MergeSchedulerConfig.MAX_THREAD_COUNT_SETTING.getKey(), 1)
+                        .put(MergeSchedulerConfig.MAX_MERGE_COUNT_SETTING.getKey(), 1)
```

Comment:
I don't think we need to use all these settings. Can't we just ensure that there is only a single segment created after force merge or for an index?